recipes/libgpg-error/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+class GPGErrorConan(ConanFile):
+    name = "libgpg-error"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gnupg.org/software/libgpg-error/index.html"
+    topics = ("conan", "gpg", "gnupg")
+    description = "Libgpg-error is a small library that originally defined common error values for all GnuPG " \
+                  "components."
+    license = "GPL-2.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
```

Comment:
```suggestion
        "fPIC": [True, False],
```

Reply:
What does this comma serve for? "fPIC" collection is at the end, isn't it?

Reply:
For uniformity with 90% of recipes.
Without the trailing comma, code formatters, such as black, will fold this dict into one line.